docs/index.html.md

```
@@ -284,14 +284,14 @@ ArrayBuffer
 ### Diff parser
 
 ```javascript
-parseDiff(text: string): Array<{filePath, hunks}>
+parseDiff(text: string): Array<{filePath, hunks, commit}>
 ```
 
 **パースの流れ:**
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)